component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mepin/MepinTransactions.java

```diff
@@ -286,6 +330,13 @@ protected String getUserInformation(String username, String password, String acc
         } catch (IOException e) {
             throw new AuthenticationFailedException("Error while get user information from mepin ", e);
         } finally {
+            if(outputStream != null) {

```

Comment:
Can't we use try-with-resources to create outputStream? If so we don't need to close the outputStream here

Reply:
As I checked Java 8 is used from IS-5.4.0 onwards. If we use Java 8 features then this won't work with the earlier IS versions.